torch/csrc/jit/codegen/cuda/predicate_compute.cpp

```diff
@@ -8,45 +8,76 @@
 #include <torch/csrc/jit/codegen/cuda/ir_utils.h>
 #include <torch/csrc/jit/codegen/cuda/kernel_ir_builder.h>
 #include <torch/csrc/jit/codegen/cuda/lower2device.h>
-#include <torch/csrc/jit/codegen/cuda/lower_utils.h>
 #include <torch/csrc/jit/codegen/cuda/transform_iter.h>
 
 namespace torch {
 namespace jit {
 namespace fuser {
 namespace cuda {
 
+namespace {
+
+// find the first (and only) TensorView output
+//
+// TODO(kir): same question as ir_utils::getTvOutput():
+//    why do we assume a single TV output?
```

Comment:
There's implicit assumptions in the approach we use that exprs only have one output TV. Haven't come across any instances yet where there would have to be multiple output TVs.

Reply:
Thinking about it, uncertain we have any exprs with more than one output.